src/queries.ts

```diff
@@ -1066,6 +1067,7 @@ export const profileNotesQuery = e.params(
         order_by: {
           expression: e.random(),
         },
+        is_owner: e.op(profile.id, '=', e.global.current_user_profile.id),
```

Comment:
As it currently is, `is_owner` is being fetched only in the profile page, but it's expected in every instance of NoteCard.

Can you move this into the query fragment `noteForCard` so that every card instance includes this data?

Reply:
Ah, and after doing this, you can remove the manual TS definition of `is_owner?: boolean | null` in the `NoteCard` type, as it'll be automatically inferred by Gel 😉